kong/pdk/log.lua

```diff
@@ -572,6 +572,13 @@ do
         response_size = tonumber(response_size, 10)
       end
 
+      local request_body
+      local response_body
+      if kong.ctx.plugin and kong.ctx.plugin.response_body then
+        request_body = kong.ctx.plugin.request_body
+        response_body = table.concat(kong.ctx.plugin.response_body)
+      end
```

Comment:
```suggestion
      local ctx_plugin = kong.ctx.plugin
      if ctx_plugin and ctx_plugin.response_body then
        request_body = ctx_plugin.request_body
        response_body = table.concat(ctx_plugin.response_body)
      end
```
There is some meta-table magic involved in getting those tables, so instead of invoking that 4 times, do it only once.

Reply:
Haha, good advice and I have fixed it. And the test result is the same as last time.

Reply:
Now all the tests have passed.